Packs/Bonusly/Integrations/Bonusly/Bonusly.py

```diff
@@ -0,0 +1,262 @@
+import json
+import math
+
+import demistomock as demisto
+import requests
+from CommonServerPython import *
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' GLOBALS/PARAMS '''
+#PARAMS = demisto.params()
+API_KEY = demisto.params().get('api_key')
+
+# Remove trailing slash to prevent wrong URL path to service
+SERVER = "http://bonus.ly"
```

Comment:
According to the docs: https://bonus.ly/api the server url should be `https`
```suggestion
SERVER = "https://bonus.ly"
```